pkg/network/tracer/connection/ebpfless_tracer.go

```diff
@@ -252,6 +250,14 @@ func (t *ebpfLessTracer) determineConnectionDirection(conn *network.ConnectionSt
 		conn.Direction = network.INCOMING
 		return
 	}
+	// only used for local connections - "outgoing" local packets
+	if conn.Dest.IsLoopback() {
```

Comment:
I don't understand this use-case. Can you please elaborate?

Reply:
This might be the wrong thing to do here to be honest. Pretty much, ebpf has the ability to figure out which direction a pre-existing connection came from, but packet capture does not. Further complicating things is, every packet shows up twice on loopback, once as incoming, once as outgoing. So I'm using this as a heuristic to decide the direction when using the loopback interface

Reply:
Actually there are still problems. I reverted `determineConnectionDirection` and will fix that in a separate PR using `SYN` (and only falling back to bound ports if it misses `SYN`)